### ISSUE TITLE
Gutenberg: v10.3.x plugin upgrade

### ISSUE THREAD

**Issue by fullofcaffeine** [Automattic/wp-calypso#51161]
Previous upgrade: https://github.com/Automattic/wp-calypso/issues/50775
Release notes: [v10.3.0](https://github.com/WordPress/gutenberg/releases/tag/v10.3.0), [v10.3.1](https://github.com/WordPress/gutenberg/releases/tag/v10.3.1), [v10.3.2](https://github.com/WordPress/gutenberg/releases/tag/v10.3.2)

### Process

- [x] Install and activate
  - [x] Copy & install v10.3.0-rc.1 
  - [x] Activate v10.3.0-rc.1 on edge 
  - [x] Copy & install v10.3.0 (D59599-code, D59601-code)
  - [x] Activate v10.3.0 on edge (D59602-code)
  - [x] Activate v10.3.0 on production (D59463-code)
  - [x] Request update for Atomic sites (p9o2xV-1kL-p2)
  - [x] Revert to v10.2.0 (D59623-code) 
  - [x] Re-activate v10.3.x on production once https://github.com/Automattic/themes/issues/3599 is fixed
  - [x] Unblock the Atomic request (p9o2xV-1kZ-p2)
  - [x] Copy & install v10.3.2 (D59858-code, D59860-code)
  - [x] Activate v10.3.2 on edge (D59861-code)
  - [x] Activate v10.3.2 on production (D59862-code)
  - [x] Request Atomic upgrade for v10.3.2 (p9o2xV-1lk-p2)
- [x] Testing
  - [x] Calypso E2E (https://github.com/Automattic/wp-calypso/pull/51525, https://github.com/Automattic/wp-calypso/pull/51752)
  - [x] Team City (D59463-code)
  - [x] Revert Calendar bug hotfix (https://github.com/Automattic/wp-calypso/pull/51625)
- [x] Publish internal announcements (p7DVsv-b3y-p2, p1617259479193200-slack-CB0B2G43X)
- [ ] Clean up unused releases
- [x] Add log entry to the Gutenberg Upgrade Log [pcoGjb-g-p2]
- [x] Open a new issue for the next upgrade, transfer remaining tasks
- [ ] Close this issue.

### Blockers 🙅‍♀️

- [x] Broken themes' properties that use experimental features
  - Reported in: https://github.com/Automattic/themes/issues/3599
  - Slack context: p1617267873288500-slack-C7YPUHBB2
  - Caused a revert to v10.2 (D59623-code)
  - Fixed in v10.3.1

### Other issues 🐛

- [x] Default font for non-FSE themes missing
  - Reported in https://github.com/Automattic/themes/issues/3280
  - Upstream fix: https://github.com/WordPress/gutenberg/pull/30080 (will be included with v10.3.0)
  - Hotfixed on Simple sites via D59109-code
  - Slack context: p1616412181032500-slack-C7YPUHBB2
- [x] Opting out of the widgets-block-editor theme support crashes the editor serverside (blocker, it crashes the editor and frontend on WPCOM)
  - Originally reported on Slack in: p1616697969125400-slack-C7YPUHBB2
  - Issue: https://github.com/WordPress/gutenberg/issues/30305 (thanks @draganescu!)
  - Fix: https://github.com/WordPress/gutenberg/pull/30318 (thanks @noisysocks!)
  - Hotfixed in 10.3.0-rc.1 in the activation diff, will be included in the 10.3.0 stable release.
- [x] `@`-mentions broken when more than one used inline
  - Reported in p1617641512194600-slack-C010KDAPG49
  - Fixed in https://github.com/WordPress/gutenberg/pull/30540
  - Fix will be included in v10.3.2

---
FYI: @Automattic/team-calypso

**Comment by WunderBart** [comment]
Before the next (v10.4) upgrade: Make sure to test https://github.com/WordPress/gutenberg/pull/30231. Context: p1616675249116100-slack-C7YPUHBB2. cc @simison

**Comment by WunderBart** [comment]
Due to a regression in v10.3 that affected 9 themes, we've decided to revert production to v10.2. See p1617267873288500-slack-C7YPUHBB2 for more info.

**Comment by WunderBart** [comment]
Another issue with v10.3.2 that causes OOM errors on P2 posts where the `Latest Posts` block was used (FYI that block has been permanently disabled some time ago, so that might be the cause of this issue). We were able to determine that this is a P2-only issue and happens only for the v10.3.2 release. Both previous v10.3.1, and next v10.4.0-rc.1 does not seem to cause this issue. Context:
- p9oQ9f-th-p2
- p1617880554267800-slack-C010KDAPG49

**Comment by WunderBart** [comment]
Next one: https://github.com/Automattic/wp-calypso/issues/51767
